sdk/cosmos/azure-cosmos/tests/test_cosmos_http_logging_policy.py

```diff
@@ -87,20 +89,19 @@ def test_default_http_logging_policy(self):
         self.client_default.delete_database(database_id)
 
     def test_cosmos_http_logging_policy(self):
-        # Test if we can log into from creating a database
+        # Test if we can log into from reading a database
         database_id = "database_test-" + str(uuid.uuid4())
         self.client_diagnostic.create_database(id=database_id)
         assert all(m.levelname == 'INFO' for m in self.mock_handler_diagnostic.messages)
-        messages_request = self.mock_handler_diagnostic.messages[13].message.split("\n")
-        messages_response = self.mock_handler_diagnostic.messages[14].message.split("\n")
-        elapsed_time = self.mock_handler_diagnostic.messages[5].message.split("\n")
-        assert "/dbs" in messages_request[0]
-        assert messages_request[1] == "Request method: 'POST'"
-        assert 'Request headers:' in messages_request[2]
-        assert messages_request[15] == 'A body is sent with the request'
-        assert messages_response[0] == 'Response status: 201'
-        assert "Elapsed time in seconds:" in elapsed_time[0]
-        assert "Response headers" in messages_response[1]
+        messages_request = self.mock_handler_diagnostic.messages[1]
+        messages_response = self.mock_handler_diagnostic.messages[2]
+        elapsed_time = messages_request.duration
+        assert "databaseaccount" == messages_request.resource_type
+        assert messages_request.verb == "GET"
+        assert 200 == messages_request.status_code
+        assert "Read" == messages_request.operation_type
+        assert elapsed_time is not None
+        assert "Response headers" in messages_response.message
```

Comment:
this is all changed because we do the initial background call to set up the database account - however since we are also running the `create_database` method, would it be possible to extend the test to also check those entries as opposed to only checking the inital read database call that happens in the background?